phpunit.xml.dist

```diff
@@ -11,19 +11,11 @@
          syntaxCheck="false"
```

Comment:
as `phpunit.xml.dist` file is managed by `dev-kit`, I am not sure what will survive after the next run

Reply:
no it isn't :-)

Reply:
https://github.com/sonata-project/dev-kit/blob/master/config/projects.yml#L189

Reply:
You are right 👍